### DISCUSSION TITLE
许多问题+111111

### DISCUSSION THREAD

**Discussion by Chengwang-an** [WeNeedHome/SummaryOfLoanSuspension#701] [General]
首先，利益无关。单纯只是觉得觉得这件事情很严重，来凑凑热闹。当然，我非常同情买到烂尾房的同志们，我也认为断贷是很合理的行为。但是有几个问题，我看没人讨论啊，我就问下。
1.这次断贷截止到目前总共有200多个楼盘，一个楼盘按2000人算。各个小区的声明是否都是全体业主共同参与发表的。如果只是烂尾楼少数业主，那真的是雷声大雨点小，最后不会获得公平待遇的。而且带头的人喝茶是避免不了的。即使真的大家都宣称自己断贷了，都多少人到时候真敢不还贷款，上征信？毕竟像我这样喊口号，电脑码字是容易的，真干起来可就不一样了。
2.关于这次断贷的目的，我想很简单，就是各位业主能拿到本该属于他们的房子。但是烂尾楼工程真的是很麻烦的事情，里面牵扯到开发商，政府，各类建筑商，材料商，农民工等，一堆三角债，资金缺口。有多少企业能有接盘的能力？况且这次还是全国各地“多点开花”。理论上说，这不仅要耗费钱，还要耗费大量的精力。毕竟冰冻三尺，非一日之寒。再加上，现在是疫情时代，国内的防疫政策已经给财政造成了极大的负担，政府掏这个钱我想应该是及不情愿的。这种费钱，费力，补窟窿，不讨好的事情。你们觉得政府真的会妥善解决么？事实上，我估计以政府的做法，大概是采用怀柔政策，分而治之。一边搞点无息贷款，一边先修一些完工程度比较高的烂尾楼，降低下民众的怒气。等到事情热度降了，事情就很难处理。这个可以参考郑州银行的案例。
        综上，这次这个事件，到底有多大的可能性让政府给房主们一个公平的交代？这是一个很复杂的问题。。。哎，道阻且长，但愿行则将至。

**Comment by hdp521** [comment]
车都倒了 你还想看前面的风景 哈哈哈

**Reply by Chengwang-an** [reply]
如果真的那些业主都断贷，那这件事情就还没有结束

**Comment by dalezhang93** [comment]
不反抗肯定是不会解决的, 这还是烂尾的楼盘, 交房了也有不少断供的, 经济下行,失业人数增加,必然导致原有的问题暴露, 今天不解决只会引爆更大的问题, 问题不会凭空消失的

**Comment by Accelerator6666** [comment]
什么叫公平？能出现烂尾楼就已经不公平了。。。

**Comment by imhappyok** [comment]
这怎么叫费力不讨好呢？对于政府来说圆满解决这些民怨问题是政绩好吧，少修一条地铁，没收部分财产，赃款钱就来了，问题解决，到时候出个通报哪个省哪个市很好的解决了所有烂尾楼问题，让省内媒体多多报道，这不就传到中央了，作为地方政府领导人，在老百姓的口碑就起来了，简直是白送的立功机会

**Reply by daiyingwo** [reply]
还是赵家人的思维
